clients/drcachesim/simulator/cache_fifo.h

```diff
@@ -54,12 +54,11 @@ class cache_fifo_t : public cache_t {
     {
```

Comment:
> Small fixes which were not enough to justify a separate issue.

Give some context: this is large repository with many components, the dynamorio core being the main one, and this has no impact on that core.  "Small fixes in drcachesim" e.g. would go a long way.